vault/core.go

```diff
@@ -1228,8 +1228,8 @@ func NewCore(conf *CoreConfig) (*Core, error) {
 		return nil, err
 	}
 
-	// Log level
-	c.configureLogRequestLevel(conf.RawConfig.LogLevel)
+	// Log requests level
+	c.configureLogRequestsLevel(conf.RawConfig.LogRequestsLevel)
```

Comment:
This is the actual fix (supplying `LogRequestsLevel` instead of `LogLevel`), the rest of the PR is tidy up.